ale/drivers/kaguya_driver.py

```diff
@@ -0,0 +1,96 @@
+import spiceypy as spice
+import os
+import pvl
+import ale
+
+import numpy as np
+from ale.drivers.base import Driver, LineScanner, PDS3, Spice, TransverseDistortion
+
+
+class TcPds3Driver(Driver, LineScanner, PDS3, Spice, TransverseDistortion):
+    @property
+    def instrument_id(self):
+        instrument = self.label.get("INSTRUMENT_ID")
+        if instrument == "TC1":
+            return "LISM_TC1"
+        elif instrument == "TC2":
+            return "LISM_TC2"
+
+
+    @property
+    def metakernel(self):
+        self._metakernels = 'mk_kaguyatc.tm'
```

Comment:
Should this be hard coded?

Reply:
no, we need to move this somewhere common. Might as well move all the kernels as well somewhere that can be reached outside of smalls.